src/Core/View.js

```diff
@@ -79,6 +82,44 @@ function View(crs, viewerDiv, options = {}) {
     if (__DEBUG__) {
         this.isDebugMode = true;
     }
+
+    if (options.loadingScreen !== false) {
+        let loadingScreenContainer;
+        if (!options.loadingScreen || options.loadingScreen === 'itowns') {
+            const node = document.createElement('style');
+            node.innerHTML = LoadingScreenCSS;
+            document.body.appendChild(node);
+            // loading screen
+            viewerDiv.insertAdjacentHTML('beforeend', LoadingScreenHTML);
+            loadingScreenContainer = document.getElementById('itowns-loader');
+        } else if (typeof (options.loadingScreen) === 'string') {
+            loadingScreenContainer = document.getElementById(options.loadingScreen);
+        } else if (options.loadingScreen instanceof (HTMLElement)) {
```

Comment:
That should probably be just `Element` to make it possible to use inline SVGs (unless there are specific reasons not to support those, and require them to be embedded in a `<div>` for example)